### DISCUSSION TITLE
Questions about extensibility of osTicket

### DISCUSSION THREAD

**Discussion by ngdangtu-vn** [osTicket/osTicket#6654] [Q&A] [answered]
1. Can I modify the theme. Like create whole new theme to overwrite the default?
2. Do we have marketplace/store for plugins (& themes if yes)?

**Comment by JediKev** [comment, marked as answer]
@ngdangtu-vn 

The code is open source, so yes. You can do anything within the boundaries of the GPL license.

No, no store/marketplace. You would essentially make your own public repo and advertise on your own sites. We do not allow mod advertisement on our Forums nor here on GitHub.

Cheers.
